src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -61,16 +68,27 @@ public async Task<IActionResult> GetChannel(int id)
 
         [HttpDelete("{id}")]
         [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(Channel))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest)]
```

Comment:
These `SwaggerResponse` attributes should not be necessary. Why are you adding them?

Reply:
I'll remove them.